lib/safe-pg-migrations/base.rb

```diff
@@ -78,6 +80,7 @@ def exec_migration(connection, direction)
     end
 
     def disable_ddl_transaction
+      UselessStatementsLogger.disable_ddl_transaction if super
```

Comment:
How clean is that?

We need it because disable_ddl_transaction is executed before we inject the plugins (which are injected in the connection, we wouldn't be able to use it for this case anyway).

Reply:
Can't we monkey patch the `disable_dll_transaction!` instead?